### ISSUE TITLE
Ideas for items/monsters on the raider black market.

### ISSUE THREAD

**Issue by ExpulsionSnooper** [d3athrow/vgstation13#32695]
[discussion]

The tl;dr is that for raiders (specifically voxes (different race raiders soon)), I plan on adding a system that lets you sell prisoners/items on a black market, removing them from the possibility of you getting a final score bonus from them, but letting you buy nifty gadgets/ghost controlled monsters. Based on one of kuroforst's ideas for raiders. Please, tell me YOUR ideas. So far, I've come up with a water cooler that dispenses chloral hydrate, and sleep toxin, and shock belts to keep prisoners in line (shock collars are too fetishy imo).
As for monsters, something along the lines of a goliath covered in mirrors to reflect laser shots is what I'm looking for (another koroforest idea).

**Comment by BomberBro** [comment]
\>Black Market
![nostalgia](https://user-images.githubusercontent.com/51391726/170814562-fc164c2d-09f5-435f-8d9f-d0043db41af7.jpg)
Otherwise maybe something like a space shark, space gorillas that can use melee weapons, or a hivelord that you can use as a gun to shoot the little hivelings at people (only slightly less of a blatant ripoff of Half-Life than the current one)

**Comment by cadaverouscoyote** [comment]
how about like a pAI/borer thing that you can forcibly jack into an existing simple mob to put it under player control?
and how about some kind of space anaconda that can slither through vents and coil around victims to cuff & slow them, while being vulnerable to attack as it's constricting?
and how about a sort of living, ambulatory space hornet nest, like an infested diona under their control or something?
also just make it a collar. as a belt it wouldn't be distinct enough from an electropack, or even preferable to it, what with it falling off if you strip a prisoner's jumpsuit

**Comment by tuypo1** [comment]
Good call on the no collars

**Comment by pedr69** [comment]
>shock collars

we already have those and they're called electropacks

**Comment by Exxion** [comment]
I maintain that the original black market had potential and was just implemented really badly
Also I still don't like raiders having capture objectives
The Inviolate was what made them unique as antags and also it's extremely not fun to get locked in an inescapable jail by antags who spawn with the best sleep weapon in the game

**Comment by ExpulsionSnooper** [comment]
>Also I still don't like raiders having capture objectives  

I'm planning on getting rid of those, and switching all objectives for vox raiders to just get over a certain amount of points. I may still keep the priority department thing, though.

>also also it's extremely not fun to get locked in an inescapable jail by antags who spawn with the best sleep weapon in the game

I'm hoping the black market teleporter partially remedies this by giving prisoners a "way out", not in the way of it letting escape it by ghosting, but actually sending the player back to the title screen, basically allowing them to respawn or observe if they still want to do that. Another change I plan on making that will go hand in hand with this is some kind of crank/machine/treadmill that prisoners interact with/use to increase how much time the raiders can stay for. (another kuruforest idea). I'm hoping that whatever I make is engaging enough for people to not ghost immediately, though, I don't think people usually do that when they're captured.

>we already have those and they're called electropacks

Sadly, they wouldn't work as they currently are, since ANYBODY can remove an electropack from someone else, so that makes handling more than one prisoner impossible. This does make me think that it might just be a good idea to overhaul electropacks, possibly instead of using signalers some unique remote that you can link with electropacks that can just flip through the options of the packs that are linked, and let you remotely detach them too. The other option is instead of a belt, it could be a shock helmet. It would fix the problems that cadaverouscoyote listed, while still having an identifiable look to it.

**Comment by WindowsErrors** [comment]
what was the original black market like

**Comment by cadaverouscoyote** [comment]
>overhaul electropacks, possibly instead of using signalers some unique remote

oh fuck no, that'd close off a ton of possibilities with them, e.g. desk bells, prox. sensors, voice analyzers. if you're gonna do an electropack remote, it should be as a supplement to signalers and not a replacement
>The other option is instead of a belt, it could be a shock helmet

or a regular old shock collar because people don't normally see a common sci-fi cliche & think "deviantart fetish fuel"
but, if you are genuinely too icked out to consider that as an option: then honestly it's a you problem. there's an abundance of so much more egregious shit on this server-- gender tf, furry tf, animal tf, clown tf, french maids, hard vore, soft vore, brainwashing, bootyborgs, hyper-farting, weight gain, exploding from overeating, stomping on small rodents-- that your hang-up on this particular, comparatively innocuous point is frankly bizarre
but if you absolutely need an alternative: an ear-slot item. kidnappers take the headset off first thing anyway. it can be like a sonic device that stuns the victim when triggered by a spoken command in vox pidgin or by an attempt to remove it

**Comment by ExpulsionSnooper** [comment]
I just don't want to be called a pervert. Honestly, the issue of where the stunning thing goes is really not the most important thing, anyways, since if the code for the thing is good enough, I should be able to quickly and easily change where/how it's attached.

**Comment by tuypo1** [comment]
>we have all this other fetish shit so that makes it ok to add more

no, fuck all this focus on gags right now as well

**Comment by cadaverouscoyote** [comment]
>I just don't want to be called a pervert.

why even call attention to it at all in the first place?

>>we have all this other fetish shit so that makes it ok to add more
>>no, fuck all this focus on gags right now as well

alright i'm beginning to get the sinking suspicion that those who see fetish shit everywhere they look yet aren't able to laugh with everyone else at the inherent absurdity of it are the ones feeling like they're the butt of the joke :|
you guys are in the <1% of people whose brain goes to this place first

**Comment by tuypo1** [comment]
Yes exactly, I think collars are sexy, that's why I can recognize it doesn't belong in a game that should be avoiding sexual content, this isn't some niche paraphilia bondage fetishes are common.

**Comment by cadaverouscoyote** [comment]
it's not sexual content. you're greatly overestimating the number of people who pop a boner when medbay straitjackets them, even among the bdsm fetishists here. the average player's able to establish a disconnect between their personal fetishes & their 2d spacemen and not drag others into their magical realm; we wouldn't be able to have xenomorphs at all if that weren't the case

**Comment by WindowsErrors** [comment]
damn what happened to this discussion, I look away for a single day and y'all've huffed the BDSM fumes harder than I have, at least I've learned to keep it in my pants over the years

this reminds me of when I tried to get into Citadel station before it died, I could only stomach it for about a week because literally every single discussion would cycle back to this without fail, any long-term discussion had a half-life of 3 days because that was the average time period before it would get hijacked by everyone trying (and oftentimes failing) to find a way to get it to work with the ERP code

Bomberbro, notify the hazardous materials team, we've got potential erotic contamination on the github

**Comment by ExpulsionSnooper** [comment]
>why even call attention to it at all in the first place?

I was just giving some reasoning as to why I wanted to go for the non"obvious" option. This is not that big of a deal, so please, just give me some cool machine/item/monster ideas, I beg of you.

**Comment by cadaverouscoyote** [comment]
as long as you understand that you're being a weirdo
a street organ that entrances nearby monkeys with its song, making them follow the organ grinder and attack any crew members they see by violently latching onto them like slimes & biting repeatedly. the aggression behavior could last for a short duration after they leave the active organ's radius or for as long as they remain latched onto a victim, and the following behavior would last for a longer duration outside of the radius than the aggression lasted, so that they'd attempt to return after running their target off of the organ grinder, but fall out of the trance if they can't reach it in time. if the organ grinder accepts a coin or credits handed to them by a crew member, any entranced monkeys will no longer target that crew member

**Comment by Blithering** [comment]
A child of santa's sack but instead of presents, every time you open it there's a fresh supply of throwing bricks. 

A brain in a jar with little spider legs, can ambulate around and not much else, unless there's a headless corpse lying around, in which case it can attach and take control of the body. 

A helmet that will randomly mindswap everyone in  view when activated other than the wearer, also acts as a tinfoil helmet. Long cooldown or single use.

A single seater mini UFO, spaceworthy cart for ages 5+

The pomfinator, a mythical grenade launcher, fires eggs that transmogrify targets into chickens.

**Comment by tuypo1** [comment]
>A brain in a jar with little spider legs, can ambulate around and not much else, unless there's a headless corpse lying around, in which case it can attach and take control of the body.

I like it

**Comment by cadaverouscoyote** [comment]
>A brain in a jar with little spider legs, can ambulate around and not much else, unless there's a headless corpse lying around, in which case it can attach and take control of the body.

this would be a really good one for grayders

**Comment by Blithering** [comment]
Bank transfer valve
Ominously ticking briefcase that explodes dealing maxcap  (financial) damage to everything in the area. Anyone in dev range has their account drained of all credits and transferred to the shoal account, heavy range loses 50%, and light lose 20%.

**Comment by tuypo1** [comment]
A straightjacket could be sexy to some but it's not the first thing that comes to mind when you think of one, a collar on the other hand will bring sexual thoughts before any other uses

**Comment by cadaverouscoyote** [comment]
you gotta have a bad case of the coombrain to think bdsm before slavery

**Comment by Exxion** [comment]
>a collar on the other hand will bring sexual thoughts before any other uses

Yeah definitely not for most people
